zulipterminal/ui_tools/views.py

```diff
@@ -1939,14 +1939,21 @@ def __init__(
 
         # Get rendered message header and footer
         msg_box = MessageBox(message, controller.model, None)
+        msg_box.footer = [
+            (
+                urwid.Text(
+                    ("msg_bold", "\nPress [c] to copy message to clipboard"),
+                    align="left",
+                )
+            )
+        ]
 
         # Get raw message content widget list
-        response = controller.model.fetch_raw_message_content(message["id"])
-
-        if response is None:
+        self.response = controller.model.fetch_raw_message_content(message["id"])
```

Comment:
Since we're using this more widely now, it would be useful to give it a name which is clearer - so that we don't have to return here to look up what it means. That seems to be what you do when you use it on line ~1978?

You could also consider this change to be a refactor prep commit, since you've not updated `test_init`.

Reply:
You mean changing the variable name to something like `self.raw_message_content`? 

I'll create a different refactor PR changing the variable name and then rebase this one.